pkg/apis/config/types.go

```diff
@@ -143,3 +143,16 @@ type NodeResourceTopologyMatchArgs struct {
 	// Namespaces to be considered by NodeResourceTopologyMatch plugin
 	Namespaces []string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourceTopologyResourceAllocationScoreArgs holds arguments used to configure the ResourceAllocationScore plugin
+type NodeResourceTopologyResourceAllocationScoreArgs struct {
+	metav1.TypeMeta
+
+	KubeConfigPath          string
+	MasterOverride          string
+	Namespaces              []string
+	ScoreSchedulingStrategy string
```

Comment:
`ScoringStrategy` - to make it consistent with https://github.com/kubernetes/kubernetes/blob/ea3bcbc2050b89d11f005a2735c8808e45b8aa21/pkg/scheduler/apis/config/types_pluginargs.go#L93.

Also make it an enum, and define the possible strategies.

Reply:
Done